src/utils/referencedBy.js

```diff
@@ -0,0 +1,30 @@
+// TODO Move to cozy-client
```

Comment:
let's go for it, no?

Reply:
Yes, I will take care of it quickly.
There is a Trello card about it.